lib/eslint6-patch.js

```diff
@@ -84,21 +69,13 @@ module.exports = function patch() {
     })
   }
 
-  function patchConfigs() {
-    const originalLoadConfigData = ConfigArrayFactory.prototype._loadConfigData
-    ConfigArrayFactory.prototype._loadConfigData = function _loadConfigData() {
-      const results = originalLoadConfigData.apply(this, arguments)
-      return [...results].filter(configWithRules)
-    }
-  }
-
   function patchConfigArray() {
     const orig = ConfigArray.prototype.extractConfig
     ConfigArray.prototype.extractConfig = function extractConfig() {
       const config = orig.apply(this, arguments)
 
       RULES_BLOCKLIST.forEach((ruleName) => {
-        if (config.rules[ruleName] !== "off") {
+        if (config.rules && config.rules[ruleName] !== "off") {
```

Comment:
Seems like we can move up the `config.rules` so we avoid checking on each iteration ?